srcpkgs/woeusb/template

```diff
@@ -0,0 +1,46 @@
+# Template file for 'woeusb'
+pkgname=woeusb
+version=3.3.1
+revision=2
+wrksrc="WoeUSB-$version"
+archs=noarch
+build_style=gnu-configure
+configure_args="--with-wx-config=wx-config-gtk3"
+hostmakedepends="automake gettext libtool"
+makedepends="wxWidgets-gtk3-devel"
+depends="bash grub ntfs-3g parted wget"
+short_desc="Create a Windows USB stick installer from a real Windows DVD or image"
+maintainer="bra1nwave <bra1nwave@protonmail.com>"
+license="GPL-3.0-or-later"
+homepage="https://github.com/slacka/WoeUSB"
+distfiles="https://github.com/slacka/WoeUSB/archive/v${version}.tar.gz"
+checksum=0cab88a1113506f39d2f1c19532b2cd8d968c6a9f59129953c000e29e73f3d4f
+
+post_patch() {
+	vsed -i -e "s/@@WOEUSB_VERSION@@/$version/g" \
+		configure.ac \
+		src/woeusb src/woeusb.1 src/woeusbgui.1
```

Comment:
You can put this in `pre_configure()`

Reply:
It's technically patching steps, I would like to have something to inspect after patching.